android/app/build.gradle

```diff
@@ -57,7 +57,7 @@ android {
         applicationId "com.example.movies_and_stars"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 19
```

Comment:
I recommend 21 api version cuz, I faced a problem: "App requires Multidex support" when I tried to write long command for graddle. But up to u